lib/api/CSerializableToJson.cc

```diff
@@ -71,8 +73,8 @@ const std::string JSON_EOS_TAG{"eos"};
 std::string CSerializableToJsonStream::jsonString() const {
     std::ostringstream jsonStream;
     {
-        rapidjson::OStreamWrapper osw{jsonStream};
-        TGenericLineWriter writer{osw};
+        std::ostringstream& osw = jsonStream;
```

Comment:
Why introduce another reference variable? Just use `jsonStream` in place of `osw`.